src/app/layout.tsx

```diff
@@ -16,7 +19,48 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body>
+      <div>
+        <Card className="bg-fill bg-center text-white rounded-none" style={{ backgroundImage: "url('../images/birds.jpg')" }}>
+          <CardTitle className="text-2xl font-bold px-6 pt-12"><Image
+              src="/images/logo.png"
+              width={70}
+              height={70}
+              className={"float-left pr-2"}
+              alt="Iowa Bird Rehabilitation Logo"
+          />
+            Iowa Bird Rehabilitation</CardTitle>
+          <CardContent>
+            <div className="flex items-center text-sm text-stone-300 pb-6">
+              <span>Creating a future for our feathered friends.</span>
+            </div>
+          </CardContent>
+        </Card>
+        <div className={inter.className}>
+          {children}
+        </div>
```

Comment:
previously this was baked into BirdAlertList but now we're using layout to specify the skeleton of the page and then the page.tsx content goes here in {children}